tests/gui/pages/main.spec.js

```diff
@@ -35,6 +35,10 @@ require.extensions['.html'] = (module, filename) => {
   })
 }
 
+// NOTE(Shou): since we don't test React yet we just ignore JSX files
+// eslint-disable-next-line node/no-deprecated-api
+require.extensions['.jsx'] = _.constant(null)
```

Comment:
Why is this necessary?

Reply:
Tests explode on JSX files because they aren't handled by Webpack.